app/route_rangers_api/urls.py

```diff
@@ -1,11 +1,15 @@
 from django.urls import path
-
 from . import views
```

Comment:
This breaks for me, likely because I am running `python -m manage runserver` from `app/`. Import should probably be `from app.route_rangers_api.views import views` so you can run it from anywhere (`from route_rangers_api.views` will also work but it does not pull from the module version of this app, it pulls directly from the local file)